lib/community/digest.js

```diff
@@ -312,7 +335,7 @@ Digest.test = function (communityId, timeAmount, timeUnit, userId) {
   var now = moment()
   var then = moment().subtract(timeAmount, timeUnit)
   return Community.find(communityId).then(community => {
-    var digest = new Digest(community, then, now)
+    var digest = new Digest(community, then, now, "daily")
```

Comment:
makes sense. It's a little daunting to read ;)
I'm gonna merge this and then add a task in the backlog to refactor